### PR TITLE
Build CAPV images for Ubuntu/CentOS

### PR DESCRIPTION
**Pull request by akutz**
This patch introduces support for building CAPV images based on the process originated with the CAPA project. Images are based on Ubuntu 18.04.2 LTS and CentOS 7.

Images also include the Cloud-Init Datasource for VMware GuestInfo.

### CODE REVIEW

**Comment by akutz** [comment]
The OVAs still need to be tested by deploying them to vSphere. The testing so far includes booting the images locally on Fusion using a cloud-init ISO for the user data. The VMware GuestInfo datasource *is* detected and supported, but the OVF envelope needs to be verified with vSphere as it's not possible to do so locally.

**Comment by akutz** [comment]
The images built from this PR have been tested on vSphere. Both the CentOS 7 and Ubuntu 18.04 images work as expected.

The images for Kubernetes 1.13.6 are uploaded to the following locations:

| Image | SHA256 |
|-------|-------------|
| [CentOS 7](http://storage.googleapis.com/capv-images/release/v1.13.6/centos-7-kube-v1.13.6.ova) | `b81c2ce1e6b492f4ca1e180423d3415ebde75678e16c1930511ed0ba47d410ca` |
| [Ubuntu 18.04](http://storage.googleapis.com/capv-images/release/v1.13.6/ubuntu-1804-kube-v1.13.6.ova) | `8dc63d91a3abece8555bbc3179faf45f165b537280e2ca160bdfe27b9ce8ab9a` |

This PR is ready to be merged. The images are good to go.

File: build/images/README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+## Prerequisites
+
+### Hypervisor
+
+The images may be built using one of the following hypervisors:
+
+| OS | Builder |
+|----|---------|
+| Linux | VMware Workstation |
+| macOS | VMware Fusion |
```
Comment (frapposelli) [inline build/images/README.md]: This should work with Windows as well right?

```suggestion
| macOS | VMware Fusion |
| Windows | VMware Workstation |
```

File: build/images/cloudinit/id_rsa.capi [hunk 2]
```diff
@@ -0,0 +1,27 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```
Comment (frapposelli) [inline build/images/cloudinit/id_rsa.capi]: Perhaps we should call this key pair something like `id_rsa.insecure-capi` or something similar, to explicitly say that its private key is available on a public forum

File: build/images/README.md (hunk 1 above)
Comment (akutz) [reply build/images/README.md]: Conceivably, but none of the build tooling or paths would work with Windows today.

File: build/images/cloudinit/id_rsa.capi (hunk 2 above)
Comment (akutz) [reply build/images/cloudinit/id_rsa.capi]: Eh, I'm not overly concerned about the name. The key isn't baked into the image. If anything, we can add a section to the README in a future update to describe the test `cloudInit` directory and its contents.

File: build/images/README.md (hunk 1 above)
Comment (frapposelli) [reply build/images/README.md]: Fair enough, resolving

File: build/images/cloudinit/id_rsa.capi (hunk 2 above)
Comment (frapposelli) [reply build/images/cloudinit/id_rsa.capi]: fair enough

Comment (akutz) [reply build/images/cloudinit/id_rsa.capi]: See 79dbab555d8fc6ab39b2cf1be3339cce2f3ecabb for the doc updates mentioned above.

**Comment by akutz** [comment]
Hi @frapposelli,

How long does it usually take for these to merge once approved?

**Comment by frapposelli** [comment]
@akutz waiting on `tide` to do its job, usually it takes 30/60 minutes, this is taking longer than expected.

**Pull request merged**